src/components/HeaderLoggedIn/index.js

```diff
@@ -46,6 +52,28 @@ export default class HeaderLoggedIn extends PureComponent {
     this.setState({ isMobileMenuOpened: !isMobileMenuOpened })
   }
 
+  renderLeft = () => {
+    const { left } = this.props
+
+    if (left.type === 'back') {
+      return (
+        <button
+          className='header__left-button'
+          onClick={typeof left.action === 'function' && left.action}
```

Comment:
You are already validating the `left.action` type with the proptypes.

Try with: `onClick={Boolean(action) && action }`

Anyway, this is just a suggestion. No need to change it